### ISSUE TITLE
Creating a Docker machine fails on Windows 7

### ISSUE THREAD

**Issue by parhamdoustdar** [docker/machine#2714]
After updating to DockerToolbox-1.9.1f (I was previously using DockerToolbox-1.8.1a), `docker-machine` has stopped working. 

##Steps to reproduce

All I need to do is to simply type `docker-machine create -d virtualbox --virtualbox-memory 2048 default`.

## The result

Here is the useful bit of the output of running this in `debug` mode:

```
(default) DBG | using 192.168.99.12 for dhcp address
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list
hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            56cbe672-7f86-4863-8dc9-b81ae0fc90d3
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:edd7:d013:caef:2a98
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modif
yvm default --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadap
ter2 VirtualBox Host-Only Ethernet Adapter --cableconnected2 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modif
yvm default --natpf1 delete ssh
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | VBoxManage.exe: error: Code E_INVALIDARG (0x80070057) - One or m
ore arguments are invalid (extended info not available)
(default) DBG | VBoxManage.exe: error: Context: "RemoveRedirect(Bstr(ValueUnion.
psz).raw())" at line 1767 of file VBoxManageModifyVM.cpp
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modif
yvm default --natpf1 ssh,tcp,127.0.0.1,50376,,22
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe start
vm default --type headless
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Waiting for VM "default" to power on...
(default) DBG | VM "default" has been successfully started.
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
notifying bugsnag: [Error in driver during machine creation: exit status 1]
Error creating machine: Error in driver during machine creation: exit status 1
```

## Things I've tried

After reading another issue on Github, I removed the default machine, removed all hostonly networks, and tried to create again. I got the exact same error.

The next time I removed `~/.docker`, but still, the error persists.

My Docker-machine version is 0.5.4, and my VirtualBox version is 5.0.12.

**Comment by dgageot** [comment]
Hi @parhamdoustdar could you try Docker-machine 0.5.5 and tell us if it's different?

**Comment by parhamdoustdar** [comment]
hi,

I went to the releases page, downloaded the 1.5.5 binary, and put it where the current `docker-machine.exe` file is located. Then I verified that I'm using the right version by doing `docker-machine version`. The output was:

```
version 0.5.5, build 02c4254
```

With that done, I re-ran the previous command. I think the error hasn't changed, but I'm pasting the log here just to be sure:

```
Docker Machine Version:  0.5.5, build 02c4254
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:53004
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(flag-lookup) Calling .GetCreateFlags
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:53008
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(default) Calling .GetMachineName
(default) Calling .DriverName
(default) Calling .GetCreateFlags
(default) Calling .SetConfigFromFlags
Running pre-create checks...
(default) Calling .PreCreateCheck
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe --version
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | 5.0.12r104815
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: wmic cpu get VirtualizationFirmwareEnabled
(default) DBG | Couldn't check that VT-X/AMD-v is enabled. Will check that the vm is properly created: exit status 2147749911
(default) DBG | local Boot2Docker ISO version:  v1.9.1
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) Calling .GetConfigRaw
Creating machine...
(default) Calling .Create
(default) DBG | local Boot2Docker ISO version:  v1.9.1
(default) Copying C:\Users\Parham\.docker\machine\cache\boot2docker.iso to C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso...
(default) Creating VirtualBox VM...
(default) Creating SSH key...
(default) DBG | Creating disk image...
(default) DBG | Creating 20000 MB hard disk image...
(default) DBG | Writing magic tar header
(default) DBG | Writing SSH key tar header
(default) DBG | Calling inner createDiskImage
(default) DBG | &{C:\Program Files\Oracle\VirtualBox\VBoxManage.exe [C:\Program Files\Oracle\VirtualBox\VBoxManage.exe convertfromraw stdin C:\Users\Parham\.docker\machine\machines\default\disk.vmdk 20971520000 --format VMDK] []  <nil> <nil> <nil> [] <nil> <nil> <nil> <nil> false [] [] [] [] <nil>}
(default) DBG | Starting command
(default) DBG | Copying to stdin
(default) DBG | Filling zeroes
(default) DBG | Closing STDIN
(default) DBG | Waiting on cmd
(default) DBG | Converting from raw image file="stdin" to file="C:\Users\Parham\.docker\machine\machines\default\disk.vmdk"...
(default) DBG | Creating dynamic image with size 20971520000 bytes (20000MB)...
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe createvm --basefolder C:\Users\Parham\.docker\machine\machines\default --name default --register
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Virtual machine 'default' is created and registered.
(default) DBG | UUID: 28b14a61-0388-4967-a89c-d800367fb2f0
(default) DBG | Settings file: 'C:\Users\Parham\.docker\machine\machines\default\default\default.vbox'
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | VM CPUS: 1
(default) DBG | VM Memory: 2048
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --firmware bios --bioslogofadein off --bioslogofadeout off --bioslogodisplaytime 0 --biosbootmenu disabled --ostype Linux26_64 --cpus 1 --memory 2048 --acpi on --ioapic on --rtcuseutc on --natdnshostresolver1 off --natdnsproxy1 off --cpuhotplug off --pae on --hpet on --hwvirtex on --nestedpaging on --largepages on --vtxvpid on --accelerate3d off --boot1 dvd
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic1 nat --nictype1 82540EM --cableconnected1 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | using 192.168.99.6 for dhcp address
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter --cableconnected2 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storagectl default --name SATA --add sata --hostiocache on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storageattach default --storagectl SATA --port 0 --device 0 --type dvddrive --medium C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storageattach default --storagectl SATA --port 1 --device 0 --type hdd --medium C:\Users\Parham\.docker\machine\machines\default\disk.vmdk
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe guestproperty set default /VirtualBox/GuestAdd/SharedFolders/MountPrefix /
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe guestproperty set default /VirtualBox/GuestAdd/SharedFolders/MountDir /
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | setting up shareDir
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe sharedfolder add default --name c/Users --hostpath c:\Users --automount
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe setextradata default VBoxInternal2/SharedFoldersEnableSymlinksCreate/c/Users 1
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo default --machinereadable
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | name="default"
(default) DBG | groups="/"
(default) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)"
(default) DBG | UUID="28b14a61-0388-4967-a89c-d800367fb2f0"
(default) DBG | CfgFile="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\default.vbox"
(default) DBG | SnapFldr="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\Snapshots"
(default) DBG | LogFldr="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\Logs"
(default) DBG | hardwareuuid="28b14a61-0388-4967-a89c-d800367fb2f0"
(default) DBG | memory=2048
(default) DBG | pagefusion="off"
(default) DBG | vram=8
(default) DBG | cpuexecutioncap=100
(default) DBG | hpet="on"
(default) DBG | chipset="piix3"
(default) DBG | firmware="BIOS"
(default) DBG | cpus=1
(default) DBG | pae="on"
(default) DBG | longmode="on"
(default) DBG | cpuid-portability-level=0
(default) DBG | bootmenu="disabled"
(default) DBG | boot1="dvd"
(default) DBG | boot2="dvd"
(default) DBG | boot3="disk"
(default) DBG | boot4="none"
(default) DBG | acpi="on"
(default) DBG | ioapic="on"
(default) DBG | biossystemtimeoffset=0
(default) DBG | rtcuseutc="on"
(default) DBG | hwvirtex="on"
(default) DBG | nestedpaging="on"
(default) DBG | largepages="on"
(default) DBG | vtxvpid="on"
(default) DBG | vtxux="on"
(default) DBG | paravirtprovider="default"
(default) DBG | VMState="poweroff"
(default) DBG | VMStateChangeTime="2015-12-31T08:40:49.463000000"
(default) DBG | monitorcount=1
(default) DBG | accelerate3d="off"
(default) DBG | accelerate2dvideo="off"
(default) DBG | teleporterenabled="off"
(default) DBG | teleporterport=0
(default) DBG | teleporteraddress=""
(default) DBG | teleporterpassword=""
(default) DBG | tracing-enabled="off"
(default) DBG | tracing-allow-vm-access="off"
(default) DBG | tracing-config=""
(default) DBG | autostart-enabled="off"
(default) DBG | autostart-delay=0
(default) DBG | defaultfrontend=""
(default) DBG | storagecontrollername0="SATA"
(default) DBG | storagecontrollertype0="IntelAhci"
(default) DBG | storagecontrollerinstance0="0"
(default) DBG | storagecontrollermaxportcount0="30"
(default) DBG | storagecontrollerportcount0="30"
(default) DBG | storagecontrollerbootable0="on"
(default) DBG | "SATA-0-0"="C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso"
(default) DBG | "SATA-ImageUUID-0-0"="e930466c-8838-4e4f-9b49-cef83e079ca9"
(default) DBG | "SATA-tempeject"="off"
(default) DBG | "SATA-IsEjected"="off"
(default) DBG | "SATA-1-0"="C:\Users\Parham\.docker\machine\machines\default\disk.vmdk"
(default) DBG | "SATA-ImageUUID-1-0"="413a4cad-47d4-45e5-8bb7-84292f60ec11"
(default) DBG | "SATA-2-0"="none"
(default) DBG | "SATA-3-0"="none"
(default) DBG | "SATA-4-0"="none"
(default) DBG | "SATA-5-0"="none"
(default) DBG | "SATA-6-0"="none"
(default) DBG | "SATA-7-0"="none"
(default) DBG | "SATA-8-0"="none"
(default) DBG | "SATA-9-0"="none"
(default) DBG | "SATA-10-0"="none"
(default) DBG | "SATA-11-0"="none"
(default) DBG | "SATA-12-0"="none"
(default) DBG | "SATA-13-0"="none"
(default) DBG | "SATA-14-0"="none"
(default) DBG | "SATA-15-0"="none"
(default) DBG | "SATA-16-0"="none"
(default) DBG | "SATA-17-0"="none"
(default) DBG | "SATA-18-0"="none"
(default) DBG | "SATA-19-0"="none"
(default) DBG | "SATA-20-0"="none"
(default) DBG | "SATA-21-0"="none"
(default) DBG | "SATA-22-0"="none"
(default) DBG | "SATA-23-0"="none"
(default) DBG | "SATA-24-0"="none"
(default) DBG | "SATA-25-0"="none"
(default) DBG | "SATA-26-0"="none"
(default) DBG | "SATA-27-0"="none"
(default) DBG | "SATA-28-0"="none"
(default) DBG | "SATA-29-0"="none"
(default) DBG | natnet1="nat"
(default) DBG | macaddress1="08002775189D"
(default) DBG | cableconnected1="on"
(default) DBG | nic1="nat"
(default) DBG | nictype1="82540EM"
(default) DBG | nicspeed1="0"
(default) DBG | mtu="0"
(default) DBG | sockSnd="64"
(default) DBG | sockRcv="64"
(default) DBG | tcpWndSnd="64"
(default) DBG | tcpWndRcv="64"
(default) DBG | hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter"
(default) DBG | macaddress2="080027C75ADE"
(default) DBG | cableconnected2="on"
(default) DBG | nic2="hostonly"
(default) DBG | nictype2="82540EM"
(default) DBG | nicspeed2="0"
(default) DBG | nic3="none"
(default) DBG | nic4="none"
(default) DBG | nic5="none"
(default) DBG | nic6="none"
(default) DBG | nic7="none"
(default) DBG | nic8="none"
(default) DBG | hidpointing="ps2mouse"
(default) DBG | hidkeyboard="ps2kbd"
(default) DBG | uart1="off"
(default) DBG | uart2="off"
(default) DBG | lpt1="off"
(default) DBG | lpt2="off"
(default) DBG | audio="none"
(default) DBG | clipboard="disabled"
(default) DBG | draganddrop="disabled"
(default) DBG | vrde="off"
(default) DBG | usb="off"
(default) DBG | ehci="off"
(default) DBG | xhci="off"
(default) DBG | SharedFolderNameMachineMapping1="c/Users"
(default) DBG | SharedFolderPathMachineMapping1="c:\Users"
(default) DBG | vcpenabled="off"
(default) DBG | vcpscreens=0
(default) DBG | vcpfile="C:\Users\Parham\.docker\machine\machines\default\default\default.webm"
(default) DBG | vcpwidth=1024
(default) DBG | vcpheight=768
(default) DBG | vcprate=512
(default) DBG | vcpfps=25
(default) DBG | GuestMemoryBalloon=0
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | using 192.168.99.12 for dhcp address
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter --cableconnected2 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 delete ssh
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | VBoxManage.exe: error: Code E_INVALIDARG (0x80070057) - One or more arguments are invalid (extended info not available)
(default) DBG | VBoxManage.exe: error: Context: "RemoveRedirect(Bstr(ValueUnion.psz).raw())" at line 1767 of file VBoxManageModifyVM.cpp
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 ssh,tcp,127.0.0.1,53016,,22
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm default --type headless
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Waiting for VM "default" to power on...
(default) DBG | VM "default" has been successfully started.
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
notifying bugsnag: [Error in driver during machine creation: exit status 1]
Error creating machine: Error in driver during machine creation: exit status 1

```

I have verified that I only have one `hostonly` network defined in VirtualBox, just to be sure.

**Comment by dgageot** [comment]
Hi @parhamdoustdar Can you try with Docker Machine 0.5.6? It should at least contain a better error message when this problem occurs.

**Comment by parhamdoustdar** [comment]
Hi,

Thanks for following up with this. The error hasn't changed as far as I can see (I'm using .06.0-rc1 build 174e390). Here is the log when running the create command in debug mode:

```
Docker Machine Version:  0.6.0-rc1, build 174e390
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:60308
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(flag-lookup) Calling .GetMachineName
(flag-lookup) Calling .DriverName
(flag-lookup) Calling .GetCreateFlags
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:60312
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(default) Calling .GetMachineName
(default) Calling .DriverName
(default) Calling .GetCreateFlags
(default) Calling .SetConfigFromFlags
Running pre-create checks...
(default) Calling .PreCreateCheck
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe --version
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | 5.0.12r104815
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: wmic cpu get VirtualizationFirmwareEnabled
(default) DBG | Couldn't check that VT-X/AMD-v is enabled. Will check that the vm is properly created: exit status 2147749911
(default) DBG | local Boot2Docker ISO version:  v1.9.1
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) Calling .GetConfigRaw
Creating machine...
(default) Calling .Create
(default) DBG | local Boot2Docker ISO version:  v1.9.1
(default) Copying C:\Users\Parham\.docker\machine\cache\boot2docker.iso to C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso...
(default) Creating VirtualBox VM...
(default) Creating SSH key...
(default) DBG | Creating disk image...
(default) DBG | Creating 20000 MB hard disk image...
(default) DBG | Writing magic tar header
(default) DBG | Writing SSH key tar header
(default) DBG | Calling inner createDiskImage
(default) DBG | &{C:\Program Files\Oracle\VirtualBox\VBoxManage.exe [C:\Program Files\Oracle\VirtualBox\VBoxManage.exe convertfromraw stdin C:\Users\Parham\.docker\machine\machines\default\disk.vmdk 20971520000 --format VMDK] []  <nil> <nil> <nil> [] <nil> <nil> <nil> <nil> false [] [] [] [] <nil>}
(default) DBG | Starting command
(default) DBG | Copying to stdin
(default) DBG | Filling zeroes
(default) DBG | Closing STDIN
(default) DBG | Waiting on cmd
(default) DBG | Converting from raw image file="stdin" to file="C:\Users\Parham\.docker\machine\machines\default\disk.vmdk"...
(default) DBG | Creating dynamic image with size 20971520000 bytes (20000MB)...
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe createvm --basefolder C:\Users\Parham\.docker\machine\machines\default --name default --register
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Virtual machine 'default' is created and registered.
(default) DBG | UUID: 6b17a971-8298-4837-bffe-ea7a71a3912c
(default) DBG | Settings file: 'C:\Users\Parham\.docker\machine\machines\default\default\default.vbox'
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | VM CPUS: 1
(default) DBG | VM Memory: 2048
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --firmware bios --bioslogofadein off --bioslogofadeout off --bioslogodisplaytime 0 --biosbootmenu disabled --ostype Linux26_64 --cpus 1 --memory 2048 --acpi on --ioapic on --rtcuseutc on --natdnshostresolver1 off --natdnsproxy1 off --cpuhotplug off --pae on --hpet on --hwvirtex on --nestedpaging on --largepages on --vtxvpid on --accelerate3d off --boot1 dvd
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic1 nat --nictype1 82540EM --cableconnected1 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storagectl default --name SATA --add sata --hostiocache on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storageattach default --storagectl SATA --port 0 --device 0 --type dvddrive --medium C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe storageattach default --storagectl SATA --port 1 --device 0 --type hdd --medium C:\Users\Parham\.docker\machine\machines\default\disk.vmdk
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe guestproperty set default /VirtualBox/GuestAdd/SharedFolders/MountPrefix /
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe guestproperty set default /VirtualBox/GuestAdd/SharedFolders/MountDir /
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | setting up shareDir
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe sharedfolder add default --name c/Users --hostpath c:\Users --automount
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe setextradata default VBoxInternal2/SharedFoldersEnableSymlinksCreate/c/Users 1
(default) Starting the VM...
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo default --machinereadable
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | name="default"
(default) DBG | groups="/"
(default) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)"
(default) DBG | UUID="6b17a971-8298-4837-bffe-ea7a71a3912c"
(default) DBG | CfgFile="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\default.vbox"
(default) DBG | SnapFldr="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\Snapshots"
(default) DBG | LogFldr="C:\\Users\\Parham\\.docker\\machine\\machines\\default\\default\\Logs"
(default) DBG | hardwareuuid="6b17a971-8298-4837-bffe-ea7a71a3912c"
(default) DBG | memory=2048
(default) DBG | pagefusion="off"
(default) DBG | vram=8
(default) DBG | cpuexecutioncap=100
(default) DBG | hpet="on"
(default) DBG | chipset="piix3"
(default) DBG | firmware="BIOS"
(default) DBG | cpus=1
(default) DBG | pae="on"
(default) DBG | longmode="on"
(default) DBG | cpuid-portability-level=0
(default) DBG | bootmenu="disabled"
(default) DBG | boot1="dvd"
(default) DBG | boot2="dvd"
(default) DBG | boot3="disk"
(default) DBG | boot4="none"
(default) DBG | acpi="on"
(default) DBG | ioapic="on"
(default) DBG | biossystemtimeoffset=0
(default) DBG | rtcuseutc="on"
(default) DBG | hwvirtex="on"
(default) DBG | nestedpaging="on"
(default) DBG | largepages="on"
(default) DBG | vtxvpid="on"
(default) DBG | vtxux="on"
(default) DBG | paravirtprovider="default"
(default) DBG | VMState="poweroff"
(default) DBG | VMStateChangeTime="2016-01-22T04:52:56.512000000"
(default) DBG | monitorcount=1
(default) DBG | accelerate3d="off"
(default) DBG | accelerate2dvideo="off"
(default) DBG | teleporterenabled="off"
(default) DBG | teleporterport=0
(default) DBG | teleporteraddress=""
(default) DBG | teleporterpassword=""
(default) DBG | tracing-enabled="off"
(default) DBG | tracing-allow-vm-access="off"
(default) DBG | tracing-config=""
(default) DBG | autostart-enabled="off"
(default) DBG | autostart-delay=0
(default) DBG | defaultfrontend=""
(default) DBG | storagecontrollername0="SATA"
(default) DBG | storagecontrollertype0="IntelAhci"
(default) DBG | storagecontrollerinstance0="0"
(default) DBG | storagecontrollermaxportcount0="30"
(default) DBG | storagecontrollerportcount0="30"
(default) DBG | storagecontrollerbootable0="on"
(default) DBG | "SATA-0-0"="C:\Users\Parham\.docker\machine\machines\default\boot2docker.iso"
(default) DBG | "SATA-ImageUUID-0-0"="01cfc8d6-22fe-4b92-9167-07003e8e9c07"
(default) DBG | "SATA-tempeject"="off"
(default) DBG | "SATA-IsEjected"="off"
(default) DBG | "SATA-1-0"="C:\Users\Parham\.docker\machine\machines\default\disk.vmdk"
(default) DBG | "SATA-ImageUUID-1-0"="0d2ac307-f820-48dc-ba57-55536e6096ac"
(default) DBG | "SATA-2-0"="none"
(default) DBG | "SATA-3-0"="none"
(default) DBG | "SATA-4-0"="none"
(default) DBG | "SATA-5-0"="none"
(default) DBG | "SATA-6-0"="none"
(default) DBG | "SATA-7-0"="none"
(default) DBG | "SATA-8-0"="none"
(default) DBG | "SATA-9-0"="none"
(default) DBG | "SATA-10-0"="none"
(default) DBG | "SATA-11-0"="none"
(default) DBG | "SATA-12-0"="none"
(default) DBG | "SATA-13-0"="none"
(default) DBG | "SATA-14-0"="none"
(default) DBG | "SATA-15-0"="none"
(default) DBG | "SATA-16-0"="none"
(default) DBG | "SATA-17-0"="none"
(default) DBG | "SATA-18-0"="none"
(default) DBG | "SATA-19-0"="none"
(default) DBG | "SATA-20-0"="none"
(default) DBG | "SATA-21-0"="none"
(default) DBG | "SATA-22-0"="none"
(default) DBG | "SATA-23-0"="none"
(default) DBG | "SATA-24-0"="none"
(default) DBG | "SATA-25-0"="none"
(default) DBG | "SATA-26-0"="none"
(default) DBG | "SATA-27-0"="none"
(default) DBG | "SATA-28-0"="none"
(default) DBG | "SATA-29-0"="none"
(default) DBG | natnet1="nat"
(default) DBG | macaddress1="080027CEDAD6"
(default) DBG | cableconnected1="on"
(default) DBG | nic1="nat"
(default) DBG | nictype1="82540EM"
(default) DBG | nicspeed1="0"
(default) DBG | mtu="0"
(default) DBG | sockSnd="64"
(default) DBG | sockRcv="64"
(default) DBG | tcpWndSnd="64"
(default) DBG | tcpWndRcv="64"
(default) DBG | nic2="none"
(default) DBG | nic3="none"
(default) DBG | nic4="none"
(default) DBG | nic5="none"
(default) DBG | nic6="none"
(default) DBG | nic7="none"
(default) DBG | nic8="none"
(default) DBG | hidpointing="ps2mouse"
(default) DBG | hidkeyboard="ps2kbd"
(default) DBG | uart1="off"
(default) DBG | uart2="off"
(default) DBG | lpt1="off"
(default) DBG | lpt2="off"
(default) DBG | audio="none"
(default) DBG | clipboard="disabled"
(default) DBG | draganddrop="disabled"
(default) DBG | vrde="off"
(default) DBG | usb="off"
(default) DBG | ehci="off"
(default) DBG | xhci="off"
(default) DBG | SharedFolderNameMachineMapping1="c/Users"
(default) DBG | SharedFolderPathMachineMapping1="c:\Users"
(default) DBG | vcpenabled="off"
(default) DBG | vcpscreens=0
(default) DBG | vcpfile="C:\Users\Parham\.docker\machine\machines\default\default\default.webm"
(default) DBG | vcpwidth=1024
(default) DBG | vcpheight=768
(default) DBG | vcprate=512
(default) DBG | vcpfps=25
(default) DBG | GuestMemoryBalloon=0
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | Searching for hostonly interface for IPv4: 192.168.99.1 and Mask: ffffff00
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | Found: VirtualBox Host-Only Ethernet Adapter
(default) DBG | Removing orphan DHCP servers...
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | NetworkName:    HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | IP:             192.168.99.6
(default) DBG | NetworkMask:    255.255.255.0
(default) DBG | lowerIPAddress: 192.168.99.100
(default) DBG | upperIPAddress: 192.168.99.254
(default) DBG | Enabled:        Yes
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Name:            VirtualBox Host-Only Ethernet Adapter
(default) DBG | GUID:            aaf8a7bc-2c54-45d9-bd90-5dd03e253501
(default) DBG | DHCP:            Disabled
(default) DBG | IPAddress:       192.168.99.1
(default) DBG | NetworkMask:     255.255.255.0
(default) DBG | IPV6Address:     fe80:0000:0000:0000:2066:4463:1005:f8cc
(default) DBG | IPV6NetworkMaskPrefixLength: 64
(default) DBG | HardwareAddress: 0a:00:27:00:00:00
(default) DBG | MediumType:      Ethernet
(default) DBG | Status:          Up
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | Adding/Modifying DHCP server "192.168.99.6"...
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | NetworkName:    HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter
(default) DBG | IP:             192.168.99.6
(default) DBG | NetworkMask:    255.255.255.0
(default) DBG | lowerIPAddress: 192.168.99.100
(default) DBG | upperIPAddress: 192.168.99.254
(default) DBG | Enabled:        Yes
(default) DBG | 
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter --cableconnected2 on
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 delete ssh
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | VBoxManage.exe: error: Code E_INVALIDARG (0x80070057) - One or more arguments are invalid (extended info not available)
(default) DBG | VBoxManage.exe: error: Context: "RemoveRedirect(Bstr(ValueUnion.psz).raw())" at line 1767 of file VBoxManageModifyVM.cpp
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 ssh,tcp,127.0.0.1,60320,,22
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm default --type headless
(default) DBG | STDOUT:
(default) DBG | {
(default) DBG | Waiting for VM "default" to power on...
(default) DBG | VM "default" has been successfully started.
(default) DBG | }
(default) DBG | STDERR:
(default) DBG | {
(default) DBG | }
Error creating machine: Error in driver during machine creation: Unable to start the VM: exit status 1
notifying bugsnag: [Error creating machine: Error in driver during machine creation: Unable to start the VM: exit status 1]
```

**Comment by dgageot** [comment]
Hi @parhamdoustdar the version from master should give you the reason of the failure. Could you please try? That'd be great!

**Comment by parhamdoustdar** [comment]
Of course. However, do I need to build it myself, or can I download it 
as a release?

**Comment by dgageot** [comment]
Sorry @parhamdoustdar for now, you have to build it yourself...
